src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
 -   **javaOptions** - Java options appended to the command used to
     start the node on the remote host.
 
+-   **deploymentMode** - Specifies how the ProActive node command is started.
+The deploymentMode can take the following values:
+
+*** _autoGenerated_: when this mode is selected the command to start the
+ProActive node will be generated automatically. As a result, the ssh call to
+the nodes will attach the nodes to the infrastructure without modifications to
```

Comment:
What does `attach the nodes to the infrastructure` mean? Does it mean starting Proactive nodes on the infrastructure?